python_testcontainers/infrahub_testcontainers/helpers.py

```diff
@@ -1,23 +1,61 @@
 import os
 import subprocess  # noqa: S404
 from pathlib import Path
+from typing import Optional
 
 import pytest
+from packaging.version import InvalidVersion, Version
 
-from infrahub_testcontainers import __version__ as infrahub_version
+from .container import ENV_VAR_DEFAULT_VALUES, InfrahubDockerCompose
 
-from .container import PROJECT_ENV_VARIABLES, InfrahubDockerCompose
+DEFAULT_INFRAHUB_SERVER_VERSION = "latest"
```

Comment:
This is a placeholder, I am not sure we should use `latest` here. It could be `importlib.metadata.version("infrahub-testcontainers")` instead so it corresponds to a _close_ infrahub version to the sdk one.